components/control_bar.py

```diff
@@ -24,12 +24,15 @@ def _control_item(title, title_id, item):
     """
     return dmc.Grid(
         [
-            dmc.Text(
-                title,
-                id=title_id,
-                size="sm",
-                style={"width": "100px", "margin": "auto", "paddingRight": "5px"},
-                align="right",
+            _tooltip(
```

Comment:
This one is adding that tooltip to all the instances of the `_control_item()` 🌀

Reply:
oh dang good catch I didn't realize that!! Ill fix that:)

Reply:
Fixed!